storey/table.py

```diff
@@ -1408,6 +1408,7 @@ def __init__(
         self.name = name
         self._explicit_raw_aggregations = explicit_raw_aggregations
         self._hidden_raw_aggregations = hidden_raw_aggregations
+        self._hidden_raw_aggregations.append("count")
```

Comment:
As we discussed, I think it would be better to change `SumValue`, potentially decoupling initial values from default values if necessary.

Reply:
Actually you cannot do that. Min and Max are handled at table.py:1658 :
`                    if value == math.inf or value == -math.inf:
                        value = math.nan
`
For `sum` you cannot do this trick, because the value '0' is a valid value

Reply:
But you raised a good point - at the bug description the min and max are set to `inf` and not `nan`, means there is another path that should be handled as well. Looking into it

Reply:
I think this line should move to the caller, and there should be deduplication.

Reply:
I still think this would better be done by reworking the `AggregationValue` hierarchy, decoupling "initial value" from "default value", but it's not that straightforward and I dont' know how much more effort we want to put into this issue.